source/retryable-writes/retryable-writes.rst

```diff
@@ -762,6 +781,9 @@ cluster (mongos just forwards the error from the shard's replica set).
 Changes
 =======
 
+2019-10-21: Change the definition of "retryable write" to be based on the
+RetryableWriteError label.
```

Comment:
Since this spec change entails changes to driver behavior and API, I think we should bump the spec version from 1.3.1 to 1.4.0.